packages/ipfs-topology/src/ipfs-topology.ts

```diff
@@ -111,8 +111,16 @@ export class IpfsTopology {
     ipfs: IpfsApi,
     bootstrapList: string[]
   ): Promise<void> {
+    // Don't want to swarm connect to ourself
+    const myPeerId = (await ipfs.id()).id
+    const filteredBootstrapList = bootstrapList.filter((addr) => {
+      const arr = addr.split('/')
+      const peerId = arr[arr.length - 1]
+      return myPeerId != peerId
```

Comment:
Can replace this with `endsWith` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith

Reply:
oh yeah, that's much nicer, thanks!